pandas/core/frame.py

```diff
@@ -5168,9 +5168,8 @@ def _get_agg_axis(self, axis_num):
 
     def mode(self, axis=0, numeric_only=False):
         """
-        Gets the mode(s) of each element along the axis selected. Empty if
```

Comment:
yes you woould move ``def mode`` to ``generic.py`` (leaving a stub here and in series for an ``@Appender`` doc-string (look at virtually any other doc-string, e.g. ``fillna``)

can do this as a follow up though (or here, up 2 u)